src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people
+    .filter((i) => i.sex === 'm')
+    .map((person) => person.died - person.born);
+
+  if (century !== undefined) {
+    men = people
+      .filter((i) => i.sex === 'm' && century === Math.ceil(i.died / 100))
+      .map((person) => person.died - person.born);
+  }
```

Comment:
```suggestion
const men = people.filter(person =>
    century
      ? Math.ceil(person.died / 100) === century && person.sex === 'm'
      : (person.sex === 'm')
    )
```
Це все можна об'єднати і записати в одному методі filter. 
До того ж, в умові сказано, що використовувати if у цьому завданні не можна